deploy/cephcsi/image/Dockerfile

```diff
@@ -48,4 +48,7 @@ LABEL maintainers="Ceph-CSI Authors" \
 
 COPY --from=builder ${SRC_DIR}/_output/cephcsi /usr/local/bin/cephcsi
 
+# verify that all dynamically linked libraries are available
+RUN [ $(ldd /usr/local/bin/cephcsi | grep -c '=> not found') = '0' ]
```

Comment:
@nixpanic do we need this check as we are building the go binary with the same base image. won't the go build will fail if the libraries are not available?

Reply:
The same base image, but we do install additional tools/libraries in the builder container. It is unlikely that this check will fail, but in case it does, we need to get informed.